<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-3GNSTIR" failures="0" tests="13" name="tests.CreateAccountTest" time="115.747" errors="1" timestamp="2022-05-25T23:18:10 EET" skipped="0">
  <testcase classname="tests.CreateAccountTest" name="HomePageSignIn" time="9.440"/>
  <system-out/>
  <testcase classname="tests.CreateAccountTest" name="tearDown" time="3.119">
    <error message="Tried to run command without establishing a connection
Build info: version: &#039;4.1.4&#039;, revision: &#039;535d840ee2&#039;
System info: host: &#039;DESKTOP-3GNSTIR&#039;, ip: &#039;192.168.192.1&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;17.0.2&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Command: [0e138063-fd23-4d85-a914-166eceee2794, close {}]
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 98.0.2, javascriptEnabled: true, moz:accessibilityChecks: false, moz:buildID: 20220322144853, moz:debuggerAddress: localhost:56656, moz:geckodriverVersion: 0.31.0, moz:headless: false, moz:processID: 12468, moz:profile: C:\Users\Admin\AppData\Loca..., moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, platformVersion: 10.0, proxy: Proxy(), se:cdp: ws://localhost:56656/devtoo..., se:cdpVersion: 85.0, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 0e138063-fd23-4d85-a914-166eceee2794" type="org.openqa.selenium.NoSuchSessionException">
      <![CDATA[org.openqa.selenium.NoSuchSessionException: Tried to run command without establishing a connection
Build info: version: '4.1.4', revision: '535d840ee2'
System info: host: 'DESKTOP-3GNSTIR', ip: '192.168.192.1', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.2'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Command: [0e138063-fd23-4d85-a914-166eceee2794, close {}]
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 98.0.2, javascriptEnabled: true, moz:accessibilityChecks: false, moz:buildID: 20220322144853, moz:debuggerAddress: localhost:56656, moz:geckodriverVersion: 0.31.0, moz:headless: false, moz:processID: 12468, moz:profile: C:\Users\Admin\AppData\Loca..., moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, platformVersion: 10.0, proxy: Proxy(), se:cdp: ws://localhost:56656/devtoo..., se:cdpVersion: 85.0, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 0e138063-fd23-4d85-a914-166eceee2794
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:567)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:622)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:626)
at org.openqa.selenium.remote.RemoteWebDriver.close(RemoteWebDriver.java:452)
at actions.BrowserActions.close_driver(BrowserActions.java:42)
at tests.CreateAccountTest.tearDown(CreateAccountTest.java:72)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:130)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- tearDown -->
  <system-out/>
  <testcase classname="tests.CreateAccountTest" name="HomePageSignIn" time="8.173"/>
  <system-out/>
  <testcase classname="tests.CreateAccountTest" name="HomePageSignIn" time="12.416"/>
  <system-out/>
  <testcase classname="tests.CreateAccountTest" name="HomePageSignIn" time="8.441"/>
  <system-out/>
  <testcase classname="tests.CreateAccountTest" name="HomePageSignIn" time="11.388"/>
  <system-out/>
  <testcase classname="tests.CreateAccountTest" name="HomePageSignIn" time="6.075"/>
  <system-out/>
  <testcase classname="tests.CreateAccountTest" name="HomePageSignIn" time="9.478"/>
  <system-out/>
  <testcase classname="tests.CreateAccountTest" name="HomePageSignIn" time="5.958"/>
  <system-out/>
  <testcase classname="tests.CreateAccountTest" name="HomePageSignIn" time="9.862"/>
  <system-out/>
  <testcase classname="tests.CreateAccountTest" name="HomePageSignIn" time="11.649"/>
  <system-out/>
  <testcase classname="tests.CreateAccountTest" name="HomePageSignIn" time="11.175"/>
  <system-out/>
  <testcase classname="tests.CreateAccountTest" name="HomePageSignIn" time="8.573"/>
  <system-out/>
</testsuite> <!-- tests.CreateAccountTest -->
